core/kazoo_csv/src/kz_csv.erl

```diff
@@ -88,31 +95,104 @@ fold(CSV, Fun, Acc)
 %% @doc
 %% @end
 %%--------------------------------------------------------------------
--spec take_row(csv()) -> {row(), csv()} |
-                         eof.
-take_row(<<>>) -> eof;
+-spec take_row(binary()) -> {row(), binary()} |
+                            'eof'.
```

Comment:
[This is already handled](https://github.com/2600hz/kazoo/blob/master/core/kazoo_csv/test/kz_csv_test.erl#L108)
What you are adding is allowing empty rows in THE MIDDLE of a CSV file.